python/eos/analysis.py

```diff
@@ -419,8 +419,8 @@ def sample_pmc(self, log_proposal, step_N=1000, steps=10, final_N=5000, rng=np.r
             The parameter determines the number of update steps to "look back".
             The default value of 1 disables this feature, a value of 0 means that all previous steps are used.
 
-        :return: A tuple of the parameters as array of length N = step_N * steps + final_N, the (linear) weights as array of length N, and the
-            final proposal function as pypmc.density.mixture.MixtureDensity.
+        :return: A tuple of the parameters as array of length N = step_N * steps + final_N, the (linear) weights as array of length N, the posterior values as array of length N, and the
+            final proposal function as pypmc.density.mixture.MixtureDensity.s
```

Comment:
This ``s`` is weird

Reply:
Good catch. My stupid home-office keyboard at work again.